apollo-common/src/main/java/com/ctrip/framework/apollo/common/exception/BadRequestException.java

```diff
@@ -22,11 +22,97 @@
 public class BadRequestException extends AbstractApolloHttpException {
 
   /**
-   * @see AbstractApolloHttpException#AbstractApolloHttpException(String, Object...) 
+   * @see AbstractApolloHttpException#AbstractApolloHttpException(String, Object...)
    */
   public BadRequestException(String msgtpl, Object... args) {
     super(msgtpl, args);
     setHttpStatus(HttpStatus.BAD_REQUEST);
   }
 
+  public static BadRequestException itemAlreadyExists(String itemKey) {
+    return new BadRequestException("item already exists for itemKey:%s", itemKey);
+  }
+
+  public static BadRequestException itemNotExists(long itemId) {
+    return new BadRequestException("item not exists for itemId:%s", itemId);
+  }
+
+  public static BadRequestException namespaceNotExists() {
```

Comment:
Maybe we could unify this method with `namespaceNotExists(String appId, String clusterName, String namespaceName)`?

Reply:
When refactoring the log information, I preserved the previous code semantics because of these two exceptions with response codes of 400 and 404 respectively. So I did not merge them,